app/controllers/SubmittedReturnsController.scala

```diff
@@ -44,11 +44,10 @@ class SubmittedReturnsController @Inject()(val messagesApi: MessagesApi,
                                            auditService: AuditingService)
   extends FrontendController with I18nSupport {
 
-  def submittedReturns(year: Int): Action[AnyContent] = authorisedController.authorisedAction ({ implicit request =>
+  def submittedReturns: Action[AnyContent] = authorisedController.authorisedAction ({ implicit request =>
     implicit user =>
-      if (isValidSearchYear(year)) {
         for {
-          obligationsResult <- getReturnObligations(user, year, Obligation.Status.Fulfilled)
+          obligationsResult <- getReturnObligations(user, dateService.now().getYear, Obligation.Status.Fulfilled)
```

Comment:
I've done some work on this but need more time to complete it which I don't have so will need to hand it over. I've added a commit on top called WIP which includes stuff I was trying out to fix the second tab not pulling data back but faced an issue with stripping out stuff relating to specific years because it's used in the vat return page itself (get obligation for year).

It's quite confusing and I think a refactor is needed as part of this story to avoid making it more complicated. 

Looking at the api and how vat-obligations works, it looks like we can pull back obligations without specifying to and from, just Fulfilled which is what I was trying to add and with user 999999999 I was getting 500 errors and 8] user=[] message=[VatObligationsConnector received error: The server you connecting to returned an error. [ServerSideError]- RESPONSE status: 500, body: {"code":"UNEXPECTED_JSON_FORMAT","reason":"The downstream service responded with json which did not match the expected format."}] on loading the view. 

I hope this helps - good luck.